.github/workflows/stackhpc-container-image-build.yml

```diff
@@ -66,6 +66,14 @@ jobs:
           sudo ip l set dummy1 up
           sudo ip l set dummy1 master breth1
 
+      # FIXME: Without this workaround we see the following issue after the runner is power cycled:
+      # TASK [MichaelRigart.interfaces : RedHat | ensure network service is started and enabled] ***
+      #   Unable to start service network: Job for network.service failed because the control process exited with error code.
+      #   See \"systemctl status network.service\" and \"journalctl -xe\" for details.
+      - name: Kill dhclient (workaround)
+        run: |
+          sudo killall dhclient
```

Comment:
Do we need to handle the case when this process does not exist?

```
[will@juno ~]$ killall doesnotexist
doesnotexist: no process found
[will@juno ~]$ echo $?
1
```
By either adding `|| true` or allowing the step to fail?

Reply:
Good catch